### PR TITLE
test: make sure all tests are using uakt for denomination fixes #771

### LINKED ISSUE #771: adjust token unit: `uakt`
The units we store are micro-akt; akt * 10^-6.

In all config files in this repo and elsewhere (github.com/ovrclk/docs, github.com/ovrclk/operations), change the unit from `akash` to `uakt`.

@jackzampolin is the cosmos unit `ucosmos`?  Will apps "just know" how to deal with `u`?

### PR DESCRIPTION
**Pull request by hydrogen18**
Just getting rid of "akt" in the tests

### CODE REVIEW

**Review by troian — APPROVED** [review]

File: sdl/coin_test.go [hunk 1]
```diff
@@ -14,9 +14,9 @@ func TestPricing(t *testing.T) {
 		value sdk.Coin
 		err   bool
 	}{
-		{"amount: 1\ndenom: akt", sdk.NewCoin("akt", sdk.NewInt(1)), false},
-		{"amount: -1\ndenom: akt", sdk.NewCoin("akt", sdk.NewInt(1)), true},
-		{"amount: 0.7\ndenom: akt", sdk.NewCoin("akt", sdk.NewInt(1)), true},
+		{"amount: 1\ndenom: uakt", sdk.NewCoin("uakt", sdk.NewInt(1)), false},
+		{"amount: -1\ndenom: uakt", sdk.NewCoin("uakt", sdk.NewInt(1)), true},
+		{"amount: 0.7\ndenom: uakt", sdk.NewCoin("uakt", sdk.NewInt(1)), true},
```
Comment (boz) [inline sdl/coin_test.go:19]: `testutil.CoinDenom`

**Review by boz — COMMENTED** [review]
use `testutil.CoinDenom` wherever you can.

if the event tests _need_ to be in the `types` package and that would cause an import cycle then no problem.

File: sdl/coin_test.go (hunk 1 above)
Comment (Ropes) [reply sdl/coin_test.go:19]: :+1:

**Review by Ropes — APPROVED** [review]
Thanks! :+1:  to using `testutil.CoinDenom`

**Pull request merged**